cmd/wallet-web/src/pages/Signup.vue

```diff
@@ -148,20 +155,30 @@ export default {
     isLoggedIn() {
       return this.isUserLoggedIn();
     },
+    isSuspended() {
+      return this.isLogInSuspended();
```

Comment:
`isLoginSuspended` would be more precise, as `login` is a noun here

Reply:
updated